hstream/app/client.hs

```diff
@@ -190,30 +191,38 @@ getNodes CliConnOpts{..} =
 
 -- FIXME: Currently, every new command will create a new connection to a server,
 -- and this needs to be optimized. This could be done with a grpc client pool.
-interactiveSQLApp :: HStreamSqlContext -> IO ()
-interactiveSQLApp ctx@HStreamSqlContext{..} = do
+interactiveSQLApp :: HStreamSqlContext -> Maybe FilePath -> IO ()
+interactiveSQLApp ctx@HStreamSqlContext{..} historyFile = do
   putStrLn helpInfo
   tid <- myThreadId
   void $ forkFinally maintainAvailableNodes (\case Left err -> throwTo tid err; _ -> return ())
-  H.runInputT H.defaultSettings loop
+  RL.runInputT settings loop
   where
+    settings :: RL.Settings IO
+    settings = RL.Settings RL.noCompletion historyFile False
```

Comment:
Why set `autoAddHistory` to False?

Reply:
because if input is split into multiple lines, it's hard to search and review history for executed statements. it will commit to history in the following cases:
- normal simple one line input
- multiple lines input, and read to SQL with delimited symbol
- multiple input, invalid but user commit as it is with Ctrl-D